packages/core/src/scope.ts

```diff
@@ -0,0 +1,126 @@
+import { Scope as BaseScope } from '@sentry/shim';
+import { Breadcrumb, SentryEvent, User } from '@sentry/types';
+
+/** An object to call setter functions on to enhance the event */
+export class Scope implements BaseScope {
```

Comment:
Rename this class to something like `BaseScope` or `ScopeImpl` to distinguish it from the interface. Otherwise code using either of the two might become confusing.